CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## UNRELEASED
 
+FEATURES:
+* Connect: Add a security context to the init copy container and the envoy sidecar and ensure they do not run as root. [[GH-493](https://github.com/hashicorp/consul-k8s/pull/493)]
+
+BUG FIXES:
+* Connect: Use `RunAsNonRoot: false` for connect-init's container when tproxy is enabled. [[GH-493](https://github.com/hashicorp/consul-k8s/pull/493)]
+
```

Comment:
I think we should also call out that we will error out if you have the same user as envoy as a breaking change. Unless we want to error out only when tproxy is enabled.

Reply:
Yeah good point. But now I have a FEATURE + BREAKING CHANGE + BUG FIX entry in the changelog against the same PR, can we consolidate this somehow?

Reply:
Haha, yes that's true. Maybe change the non-bug fix to a breaking change and add a note about what's breaking.

Reply:
@ishustava  how does this look?

Reply:
Perfect!